test/seed.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Test Playbook
+  hosts: seed-hosts
+  vars:
+    ci_cd:
+      namespace_1: applier-ci-1
+      namespace_2: applier-ci-2
+    secret_params:
+      username: testUsername
+      password: testPassword
+  tasks:
+    - include_role:
+        name: "{{ playbook_dir }}/../roles/openshift-applier"
+    - name: Delete projects that were created from this test
```

Comment:
Would suggest doing something like the following to test the `openshift-applier` to its fulles:

```
    - include_role:
        name: "{{ playbook_dir }}/../roles/openshift-applier"
      vars:
         provision: false
```

Reply:
Using `provision: false` does not work, there are permission issues (see below). Using `oc delete` still works, leaving this where it is
```
"Error from server (Forbidden): projectrequests \"applier-ci-1\" is forbidden: User \"pcarney\" cannot delete projectrequests at the cluster scope: 
```